dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -532,6 +532,9 @@ public enum Status {
     QUERY_PROCESS_DEFINITION_ALL_VARIABLES_ERROR(1300100, "query process definition all variables error",
             "查询工作流自定义变量信息错误"),
 
+    BATCH_RELEASE_PROCESS_DEFINE_BY_CODE_STATES_ERROR(1300101, "batch release process definition by code states error",
+            "批量释放工作流定义失败"),
```

Comment:
It is better to change it to  ``` "批量{0}工作流失败" ```  Whether it is online or offline, we can get it through the parameters of the interface